cmd/scollector/collectors/snmp.go

```diff
@@ -67,3 +91,62 @@ func snmp_oid(host, community, oid string) (*big.Int, error) {
 	err := snmp.Get(host, community, oid, &v)
 	return v, err
 }
+
+func c_snmp_generic(cfg conf.SNMP, mib conf.MIB) (opentsdb.MultiDataPoint, error) {
+	md := opentsdb.MultiDataPoint{}
+	baseOid := mib.BaseOid
+
+	for _, metric := range mib.Metrics {
+		rate := metadata.RateType(metric.RateType)
+		if rate == "" {
+			rate = metadata.Gauge
+		}
+		unit := metadata.Unit(metric.Unit)
+		if unit == "" {
+			unit = metadata.None
+		}
+		tagset := opentsdb.TagSet{"host": cfg.Host}
+		v, err := snmp_oid(cfg.Host, cfg.Community, baseOid+metric.Oid)
```

Comment:
My understanding of the spec was that if `metric.Oid` starts with ` .`, then use base. Otherwise don't use base. Is this still true?